sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3266,16 +3266,24 @@ _ZN2cl4sycl6mallocEmRKNS0_6deviceERKNS0_7contextENS0_3usm5allocE
 _ZN2cl4sycl6streamC1EmmRNS0_7handlerE
 _ZN2cl4sycl6streamC2EmmRNS0_7handlerE
 _ZN2cl4sycl7contextC1EP11_cl_contextSt8functionIFvNS0_14exception_listEEE
-_ZN2cl4sycl7contextC1ERKNS0_6deviceESt8functionIFvNS0_14exception_listEEEb
-_ZN2cl4sycl7contextC1ERKNS0_8platformESt8functionIFvNS0_14exception_listEEEb
-_ZN2cl4sycl7contextC1ERKSt6vectorINS0_6deviceESaIS3_EESt8functionIFvNS0_14exception_listEEEb
-_ZN2cl4sycl7contextC1ERKSt8functionIFvNS0_14exception_listEEEb
+_ZN2cl4sycl7contextC2ERKNS0_6deviceERKNS0_13property_listE
```

Comment:
You need to increment SYCL_DEV_ABI_VERSION in sycl/CMakeLists.txt if you change ABI.
See sycl/doc/ABIPolicyGuide.md

Reply:
Thank you for pointing this out! I was wondering what the correct etiquette for changing this was. I will give it a read and update the ABI version.

Reply:
I've increased SYCL_DEV_ABI_VERSION, but I'm unsure whether or not you want me to increase the SYCL version as well?

Reply:
No, only DEV_ABI needs to be increased in such cases.

Reply:
Perfect. Thanks again for pointing me to this resource.